content/docs/guides/revocation.md

```diff
@@ -14,6 +14,61 @@ toc = false
 top = false
 +++
 
+## Why should we plan for token revocation?
+
+There are two competing approaches to session management in authorization, that will drive
+architectural decisions:
```

Comment:
```suggestion
architectural decisions:

```

Reply:
apparently zola (and now github too) can handle lists without an empty line before, and this has no effect on the rendered text

Reply:
Yeah it's an open discussion https://talk.commonmark.org/t/blank-lines-before-lists-revisited/1990/35